packages/scandipwa/src/component/CategorySort/CategorySort.component.js

```diff
@@ -79,7 +81,7 @@ export class CategorySort extends PureComponent {
               attr={ {
                   id: 'category-sort',
                   name: 'category-sort',
-                  defaultValue: `${sortDirection} ${sortKey}`,
+                  value: `${sortDirection} ${sortKey}`,
```

Comment:
The difference between the `defaultValue` and the `value` props is that the first one is used for the uncontrolled input, while the second one is for the controlled inputs.
Is the `category-sort` field controlled or uncontrolled?

Reply:
In general, there are no handlers for changing its value. But the value of sortDirection and sortKey is taken from props, and if we use defaultValue, then the value does not change depending on the opened PLP.